### PR TITLE
Add configuration updates for Tendermint v0.36.

### PR DESCRIPTION
**Pull request by creachadair**
Add several more configuration updates to the `confix` tool.
These were discovered by manual archaeology (see #8298).

Some of these fixes are specific to v0.36, some also apply to v0.35.
The latter I will combine into #8300 or backport.

### CODE REVIEW

File: scripts/confix/confix.go [hunk 1]
```diff
@@ -85,21 +85,138 @@ var plan = transform.Plan{
...
+		Desc: "Remove per-node consensus timeouts (converted to consensus parameters)",
+		T: transform.Remove(
+			parser.Key{"consensus", "skip-timeout-commit"},
+			parser.Key{"consensus", "timeout-commit"},
+			parser.Key{"consensus", "timeout-precommit"},
+			parser.Key{"consensus", "timeout-precommit-delta"},
+			parser.Key{"consensus", "timeout-prevote"},
+			parser.Key{"consensus", "timeout-prevote-delta"},
+			parser.Key{"consensus", "timeout-propose"},
+			parser.Key{"consensus", "timeout-propose-delta"},
```
Comment (williambanfield) [inline scripts/confix/confix.go:131]: Does this transformation remove the associated comments? Perhaps... should it?

**Review by williambanfield — APPROVED** [review]

File: scripts/confix/confix.go (hunk 1 above)
Comment (creachadair) [reply scripts/confix/confix.go:131]: It does. Comments are "attached" to the items they're adjacent to, so comments on an item move (or are removed) with it.

Comment (creachadair) [reply scripts/confix/confix.go:131]: (if you think we should keep the comments, that is also possible, but will require a bit of extra work)

Comment (williambanfield) [reply scripts/confix/confix.go:131]: I think it should! That seems very sensible.

**Pull request merged**